src/ngx_http_lua_socket_tcp.c

```diff
@@ -165,10 +165,10 @@ enum {
 
 
 enum {
-    SOCKET_OP_RESUME_CONN,
     SOCKET_OP_CONNECT,
     SOCKET_OP_READ,
-    SOCKET_OP_WRITE
+    SOCKET_OP_WRITE,
+    SOCKET_OP_RESUME_CONN,
```

Comment:
Can we remove the trailing comma? It's new in C99.

Reply:
@agentzh 
Comma removed.